types/tx.go

```diff
@@ -277,3 +332,115 @@ func MarshalBlobTx(tx []byte, blobs ...*tmproto.Blob) (Tx, error) {
 	}
 	return bTx.Marshal()
 }
+
+// Validate runs basic validations on the proof then verifies if it is consistent.
+// It returns nil if the proof is valid.
+// The `root` is  the block data root that the shares, to be proven, belong to.
+// Otherwise, it returns a sensible error.
```

Comment:
```suggestion
// It returns nil if the proof is valid. Otherwise, it returns a sensible error.
// The `root` is  the block data root that the shares to be proven belong to.
```